src/main/java/com/salesforce/storm/spout/sideline/trigger/example/ZookeeperWatchTrigger.java

```diff
@@ -140,7 +139,8 @@ public void open(final Map config) {
         );
 
         curator = CuratorFactory.createNewCuratorInstance(
-            Tools.stripKeyPrefix(Config.PREFIX, config)
+            Tools.stripKeyPrefix(Config.PREFIX, config),
+            "ZookeeperWatchTrigger"
```

Comment:
this.class.getSimpleName()

When you’re use a classes name, try to use this, it makes refactoring/renaming later sooo much easier.